core/src/main/scala/org/json4s/reflect/Reflector.scala

```diff
@@ -42,18 +42,26 @@ object Reflector {
     stringTypes(name, ScalaSigReader.resolveClass[AnyRef](_, ClassLoaders) map (c => scalaTypeOf(c)))
 
   def describe[T](implicit mf: Manifest[T], formats: Formats = DefaultFormats): ObjectDescriptor =
-    describe(scalaTypeDescribable(scalaTypeOf[T])(formats))
+    describe2(scalaTypeDescribable(scalaTypeOf[T]))
 
+  @deprecated("Use describe[T] or describe2 instead", "3.6.8")
   def describe(st: ReflectorDescribable[_]): ObjectDescriptor =
-    descriptors(st.scalaType, createDescriptor(_, st.paranamer, st.companionClasses))
+    descriptors(st.scalaType, createDescriptor2(_, st.paranamer, st.companionClasses)(DefaultFormats))
 
+  def describe2(st: ReflectorDescribable[_])(implicit formats: Formats): ObjectDescriptor =
```

Comment:
Can we find a better name for this `describe2`?
I know the reason for having a different name then `describe`, but this `2` looks ugly.